1 Table.sql

```diff
@@ -0,0 +1,34 @@
+use CostumesDB 
+go
+drop table if exists dbo.CostumeStore 
+go 
+
+create table dbo.CostumeStore(
+	CostumeId int not null identity primary key, 
+	CustomerName varchar(60) not null constraint ck_CostumeStore_customer_name_cannot_be_blank check(CustomerName <> ''),
```

Comment:
AS: It is always best to split up first name and last name into two separate columns, in case you need to do something specifically based off of one of them.